### PR TITLE
ADEN-9074 Load Porvata in cdm-zone-02

### PR DESCRIPTION
**Pull request by preludek**

### CODE REVIEW

File: platforms/gamepedia/setup.ts [hunk 1]
```diff
@@ -174,14 +182,35 @@ class AdsSetup {
...
 	private injectIncontentPlayer(): void {
+		const slotName = 'incontent_player';
 		const isVideo =
 			!!document.getElementById('mf-video') ||
 			!!document.getElementById('twitchnet-liveontwitch') ||
 			!!document.getElementById('ds_cpp');
 
-		if (!isVideo) {
-			slotInjector.inject('incontent_player');
+		if (isVideo) {
```
Comment (dmnsobczak) [inline platforms/gamepedia/setup.ts]: Variable `isVideo` is used only once. Maybe it's better to copy those conditions directly into if?

File: platforms/gamepedia/setup.ts [hunk 2]
```diff
@@ -174,14 +182,35 @@ class AdsSetup {
...
+			return;
+		}
+
+		if (!document.getElementById('cdm-zone-02')) {
```
Comment (dmnsobczak) [inline platforms/gamepedia/setup.ts]: Let's parametrize name `cdm-zone-02`, eg. `const porvataSlots = ['cdm-zone-02'];` + foeEach?

File: platforms/gamepedia/setup.ts [hunk 3]
```diff
@@ -174,14 +182,35 @@ class AdsSetup {
...
+			this.initiateIncontentPlayer(slotName);
 		}
+
+		slotService.on('cdm-zone-02', AdSlot.STATUS_SUCCESS, () => {
+			if (!!document.querySelector('.porvata3')) {
```
Comment (dmnsobczak) [inline platforms/gamepedia/setup.ts]: Looking for `.porvata3` in DOM is inefficient. Let's set some variable in Porvata template, eg. `context.set('options.video.porvataLoaded', true);` and use it as a condition. It's safer.

File: platforms/gamepedia/slots.ts [hunk 4]
```diff
@@ -67,6 +67,9 @@ class SlotsContext {
 			},
 			'cdm-zone-02': {
 				aboveTheFold: true,
+				disabled: true,
```
Comment (dmnsobczak) [inline platforms/gamepedia/slots.ts]: Remove this line.

File: platforms/gamepedia/setup.ts (hunk 1 above)
Comment (preludek) [reply platforms/gamepedia/setup.ts]: done

File: platforms/gamepedia/slots.ts (hunk 4 above)
Comment (preludek) [reply platforms/gamepedia/slots.ts]: done

File: platforms/gamepedia/setup.ts (hunk 3 above)
Comment (preludek) [reply platforms/gamepedia/setup.ts]: done

File: platforms/gamepedia/setup.ts (hunk 2 above)
Comment (preludek) [reply platforms/gamepedia/setup.ts]: Can you give more arguments about that change? 
I parametrized `cdm-zone-02` within the function.

File: platforms/gamepedia/setup.ts [hunk 5]
```diff
@@ -174,14 +182,36 @@ class AdsSetup {
 	}
 
 	private injectIncontentPlayer(): void {
-		const isVideo =
+		const icpSlotName = 'incontent_player';
```
Comment (fraszczakszymon) [inline platforms/gamepedia/setup.ts]: Use clear and descriptive scoped variable names please. E.g. `incontentPlayerSlotName`

File: platforms/gamepedia/setup.ts [hunk 6]
```diff
@@ -174,14 +182,36 @@ class AdsSetup {
...
+		const slotCheck = 'cdm-zone-02';
```
Comment (fraszczakszymon) [inline platforms/gamepedia/setup.ts]: Same here. E.g. `porvataAlternativeSlotName`? `porvataDisplaySlotName`?

File: platforms/gamepedia/setup.ts [hunk 7]
```diff
@@ -174,14 +182,36 @@ class AdsSetup {
...
+		});
+
+		slotService.on(slotCheck, AdSlot.STATUS_COLLAPSE, () => {
+			this.initiateIncontentPlayer(icpSlotName);
+		});
+	}
+
+	private initiateIncontentPlayer(slotName: string): void {
+		slotInjector.inject(slotName);
+		slotsContext.setState(slotName, context.get('options.video.isOutstreamEnabled'));
```
Comment (fraszczakszymon) [inline platforms/gamepedia/setup.ts]: This approach might cause race conditions once we decide to add bidders. Bidders are called based on slotsContext states and this change here may happen way too late. There is no action for now, but we'll need to remember about that during bidders integration.

File: platforms/gamepedia/setup.ts (hunk 6 above)
Comment (preludek) [reply platforms/gamepedia/setup.ts]: done

File: platforms/gamepedia/setup.ts (hunk 5 above)
Comment (preludek) [reply platforms/gamepedia/setup.ts]: done

**Pull request merged**